lib/spack/spack/cmd/external.py

```diff
@@ -42,6 +42,10 @@ def setup_parser(subparser):
         '--scope', choices=scopes, metavar=scopes_metavar,
         default=spack.config.default_modify_scope('packages'),
         help="configuration scope to modify")
+    find_parser.add_argument(
+        '-t', '--tags', action='append',
```

Comment:
This argument should be `--tag` if it only accepts a single tag, `--tags` if it accepts a comma-delimited list